### PR TITLE
global-api testing updated .env name

### PR DESCRIPTION
**Pull request by amanda-eames**
had wrong name .env file

### CODE REVIEW

**Review by evanp — CHANGES_REQUESTED** [review]
Need to fix this filename before going further

File: .github/workflows/global-api-develop.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
           sleep 10
           createuser -w -h localhost -p 5432 -U postgres citycatalyst
           createdb -w -h localhost -p 5432 -U postgres citycatalyst -O citycatalyst
-          cp env.example .env
+          cp env.sample .env
```
[the remark this answers is not in the document]
Comment (amanda-eames) [reply .github/workflows/global-api-develop.yml]: you are right!

File: .github/workflows/global-api-develop.yml [hunk 2]
```diff
@@ -11,6 +11,14 @@ on:
...
       - .github/workflows/global-api-develop.yml
     branches: ["develop"]
+  pull_request:
+    paths:
+      - global-api/**
+      - k8s/cc-global-api-deploy.yml
+      - k8s/cc-global-api.yml
+      - k8s/cc-global-api-migrate.yml
+      - .github/workflows/global-api-develop.yml
+    branches: ["develop"]
```
Comment (lemilonkh) [inline .github/workflows/global-api-develop.yml:21]: Don't need to use `develop` here, as it applies for pull requests into any branch

**Review by lemilonkh — APPROVED** [review]

File: .github/workflows/global-api-develop.yml (hunk 2 above)
Comment (lemilonkh) [reply .github/workflows/global-api-develop.yml:21]: You can just remove the branches line here

**Pull request merged**